### PR TITLE
12.0 9593791726fabafc8dbe9e3540079bc629a3c246

### PR DESCRIPTION
**Pull request by Rusllan**

### CODE REVIEW

**Comment by KolushovAlexandr** [comment]
@Rusllan I've made a commit here, dont remove it

File: portal_event_tickets/doc/changelog.rst [hunk 1]
```diff
@@ -1,3 +1,8 @@
+`1.1.0`
+-------
+
+- **NEW** New fields to show that ticket was transferred or updated
```
Comment (Rusllan) [inline portal_event_tickets/doc/changelog.rst]: Someone has to test it

File: website_event_attendee_fields_custom/doc/changelog.rst [hunk 2]
```diff
@@ -1,3 +1,8 @@
+`1.0.1`
+-------
+
+- **FIX** Updated ticket form, `as on ID` was added for name fields
```
Comment (Rusllan) [inline website_event_attendee_fields_custom/doc/changelog.rst]: Someone has to test it

File: portal_event_tickets/doc/changelog.rst (hunk 1 above)
Comment (Rusllan) [reply portal_event_tickets/doc/changelog.rst]: It is not installable. Never mind

File: website_event_attendee_fields_custom/doc/changelog.rst (hunk 2 above)
Comment (Rusllan) [reply website_event_attendee_fields_custom/doc/changelog.rst]: It is not installable. Never mind

**Comment by yelizariev** [comment]
web_login_background will be fixed later by @trojikman

**Pull request merged**